resources/intent/weedo_x40/Extrudr/weedo_x40_0.4_Extrudr_GreenTECPro_Anthracit_Extra_Fine_Engineering.inst.cfg

```diff
@@ -0,0 +1,31 @@
+[general]
+version = 4
+name = Accurate
+definition = weedo_x40
+
+[metadata]
+setting_version = 17
+type = intent
+quality_type = extra fine
+intent_category = engineering
+material = Extrudr_GreenTECPro_Black_175
```

Comment:
So the engineering intent is only there for the black material? That seems a bit weird. 

Ah. I see the issue now; the original profiles were also only there for black material, which is also weird...